layers/+tools/shell/README.org

```diff
@@ -51,10 +82,11 @@ to the following variables:
 - =term=
 - =ansi-term=
 - =multi-term=
+- =vterm=
 
 #+BEGIN_SRC emacs-lisp
   (setq-default dotspacemacs-configuration-layers
-    '((shell :variables shell-default-shell 'eshell)))
+    '((shell :variables shell-default-shell 'vterm)))
```

Comment:
```suggestion
    '((shell :variables shell-default-shell 'eshell)))
```
It might be best to leave this as `eshell` since it's probably the best supported shell.
And could you add `(default)` after `- =eshell=` in the list of possible values for `shell-default-shell`

That's what was suggested for the multiple-cursors layer.
https://github.com/syl20bnr/spacemacs/tree/develop/layers/+misc/multiple-cursors#configuration

Reply:
Haha, I am so enthusiastic about this terminal that I wanted to promote it as much as I could 😆 

After using it in Spacemacs for now over 11 month, I am convinced that it is far superior to all the other options in Emacs.

Reply:
After checking I am not sure that we should mention `(default)` beside `eshell`, since it is only the default on Windows:

```elisp
(defvar shell-default-shell (if (eq window-system 'w32)
                                'eshell
                              'ansi-term)
  "Default shell to use in Spacemacs. Possible values are `eshell' (default),
`shell', `term', `ansi-term', `multi-term' and `vterm'.")
```

Reply:
😄 I'll ask the maintainers what they think.

Reply:
I wrote it as `(default on Windows)` and `(default on macOS)`, I don't think such trivial detail is worth bothering anyone who is not already following this PR.

Reply:
I don't have much experience with shell (so you have a noob reviewing this 😄), I didn't know that the defaults were different.

That's good, It might be helpful for someone reading the readme, to know what the default value of `shell-default-shell` is if it isn't defined.